hoomd/data/__init__.py

```diff
@@ -4,6 +4,6 @@
 from .local_access import (AngleLocalAccessBase, BondLocalAccessBase,
                            ConstraintLocalAccessBase, DihedralLocalAccessBase,
                            ImproperLocalAccessBase, PairLocalAccessBase,
-                           ParticleLocalAccessBase)
-from .local_access_cpu import LocalSnapshot
-from .local_access_gpu import LocalSnapshotGPU
+                           ParticleLocalAccessBase, ForceLocalAccessBase)
+from .local_access_cpu import LocalSnapshot, ForceLocalAccess
+from .local_access_gpu import LocalSnapshotGPU, ForceLocalAccessGPU
```

Comment:
Do these belong in the this base level even though force and virial are nonsensical for HPMC. I feel like the answer is probably yes because there's lots of particle data that this is the case for.

Reply:
On second thought reading below, I think this could be an issue. I'd say this is the only thing truly blocking approval from me. The other things are clarification

Reply:
I'll move this to an md specific namespace, see my response to your next comment.